channeldb/codec.go

```diff
@@ -174,17 +174,17 @@ func writeElement(w io.Writer, element interface{}) error {
 // io.Writer using writeElement.
 func writeElements(w io.Writer, elements ...interface{}) error {
 	for _, element := range elements {
-		err := writeElement(w, element)
+		err := WriteElement(w, element)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-// readElement is a one-stop utility function to deserialize any datastructure
+// ReadElement is a one-stop utility function to deserialize any datastructure
 // encoded using the serialization format of the database.
-func readElement(r io.Reader, element interface{}) error {
+func ReadElement(r io.Reader, element interface{}) error {
```

Comment:
Should we expose the plural versions of `ReadElement` and `WriteElement` while we're at it?

Reply:
There isn't such thing 😛

Reply:
these?
https://github.com/wpaulino/lnd/blob/fdfb776007402989f2e6f0b614a91a5c8a21e1bd/channeldb/codec.go#L175
https://github.com/wpaulino/lnd/blob/fdfb776007402989f2e6f0b614a91a5c8a21e1bd/channeldb/codec.go#L355